content/acra/configuring-maintaining/general-configuration/acra-poisonrecordmaker.md

```diff
@@ -241,7 +241,7 @@ Should be provided only with `--keystore_encryption_type=<vault_master_key>` fla
 In general, this tool is used like this:
 
 1) Generate "poison keys" using one of
-   * `acra-keys generate --keystore=vX --poison_record_keys`
+   * `acra-poisonrecordmaker generate --keystore=vX --poison_record_keys`
```

Comment:
`acra-keys` doesn't generate poison record keys?

Reply:
you are right, my mistake